docs/core/linux-prerequisites.md

```diff
@@ -103,537 +105,288 @@ On Linux, there are two installer package choices:
 
 ### Scripting Installs with the .NET Core installer script
 
-The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the script from: https://dot.net/v1/dotnet-install.sh
+The [`dotnet-install` scripts](./tools/dotnet-install-script.md) are used to perform a non-admin install of the CLI toolchain and the shared runtime. You can download the script from [https://dot.net/v1/dotnet-install.sh](https://dot.net/v1/dotnet-install.sh).
 
 The installer bash script is used in automation scenarios and non-admin installations. This script also reads PowerShell switches, so they can be used with the script on Linux/OS X systems.
 
 > [!IMPORTANT]
 > Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
 
-## Install .NET Core for Red Hat Enterprise Linux (RHEL) 7
-
-To install .NET Core on RHEL 7:
-
-1. Enable the Red Hat .NET channel, available under your RHEL 7 subscription.
-    * For Red Hat Enterprise 7 Server, use:
-    
-         ```bash
-         subscription-manager repos --enable=rhel-7-server-dotnet-rpms
-         ```
-    
-    * For Red Hat Enterprise 7 Workstation, use:
-    
-        ```bash
-        subscription-manager repos --enable=rhel-7-workstation-dotnet-rpms
-         ```
-    
-    * For Red Hat Enterprise 7 HPC Compute Node, use:
-    
-        ```bash
-        subscription-manager repos --enable=rhel-7-hpc-node-dotnet-rpms
-        ```
-
-2. Install the scl tool.
-
-    ```bash
-    yum install scl-utils
-    ```
-    
-3. Install .NET Core
-
-# [.NET Core 2.x](#tab/netcore2x)
-
-Install .NET Core 2.0 SDK and Runtime:
+## Install .NET Core for supported Red Hat Enterprise Linux (RHEL) versions
 
-   ```bash
-   yum install rh-dotnet20
-   ```
+To install .NET Core on supported RHEL versions:
 
-Enable .NET Core 2.0 SDK/Runtime for your environment:
+# [.NET Core 2.x](#tab/netcore2x)
 
-   ```bash
-   scl enable rh-dotnet20 bash
-   ```
+To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxredhat) for supported RHEL versions.
```

Comment:
I think this should probably point to just the install instructions right, and not the Get Started tutorial? https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-current

Reply:
I know for most content on this page you want a specific version, but if you have places where you just want current, then you can replace the version number in the URL with `current` (like the above link)